cmd/skipper/main.go

```diff
@@ -61,6 +61,8 @@ const (
 	sourcePollTimeoutUsage         = "polling timeout of the routing data sources, in milliseconds"
 	insecureUsage                  = "flag indicating to ignore the verification of the TLS certificates of the backend services"
 	proxyPreserveHostUsage         = "flag indicating to preserve the incoming request 'Host' header in the outgoing requests"
+	idleConnsPerHostUsage          = "maximum idle connections per backend host"
+	closeIdleConnsPeriodUsage      = "period of closing all idle connections in seconds. Not closing when less than 0"
```

Comment:
i like the parse duration. I prefer the Period instead of Duration, because it is a periodic action.